x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/date_histogram.tsx

```diff
@@ -471,14 +471,14 @@ export const dateHistogramOperation: OperationDefinition<
       <>
         <p>
           {i18n.translate('xpack.lens.indexPattern.dateHistogram.autoBasicExplanation', {
-            defaultMessage: 'The auto date histogram splits a data field into buckets by interval.',
+            defaultMessage: 'Date histogram splits data into time intervals.',
           })}
         </p>
 
         <p>
           <FormattedMessage
             id="xpack.lens.indexPattern.dateHistogram.autoLongerExplanation"
-            defaultMessage="To choose the interval, Lens divides the specified time range by the {targetBarSetting} setting. Lens calculates the best interval for your data. For example 30m, 1h, and 12. The maximum number of bars is set by the {maxBarSetting} value."
+            defaultMessage="To choose the interval, Lens divides the specified time range by the {targetBarSetting} Advanced Setting and calculates the best interval for your data. For example, when the time range is 4 days, the data is divided into 12 hour intervals. To configure the maximum number of bars, use the {maxBarSetting} Advanced Setting."
```

Comment:
A time range of 4 days will result in hourly buckets with the default settings, not 12 hour intervals - the default bar target is 100 (4*24 = 96 bars with hourly buckets in the span of 4 days). Could we adjust?